dashboard/src/components/inputs/tag-input/tag-input.tsx

```diff
@@ -0,0 +1,91 @@
+import { useRef } from 'react';
+import closeIcon from '@/assets/images/dark-close.svg';
+
+const TagInput = ({
+    placeholder,
+    tags = [],
+    handleAddition,
+    handleDelete,
+}: {
+    placeholder?: string;
+    tags?: Tag[];
+    handleAddition: (tag: Tag) => void;
+    handleDelete: (id: string) => void;
+}) => {
+    const inputRef = useRef<HTMLInputElement>(null);
+    const handleKeyboardEvent = (event: React.KeyboardEvent) => {
+        if (event.key === 'Tab' && event.type === 'keydown') {
+            event.nativeEvent.stopPropagation();
+            event.nativeEvent.preventDefault();
```

Comment:
提問，這邊如果用

```suggestion
           event.stopPropagation();
            event.preventDefault();
```

會有問題嗎？想知道用 nativeEvent 的原因是什麼

Reply:
喔喔～～不會誒

Reply:
嗯嗯看起來用原本的 event 就行